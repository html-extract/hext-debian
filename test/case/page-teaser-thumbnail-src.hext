<div class="thumbnail">  <img src:image />  </div>
