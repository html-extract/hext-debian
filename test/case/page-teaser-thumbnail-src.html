<div class="thumbnail">
  <img src="image.jpg" />
</div>
<div class="nope">
  <img src="nope.jpg" />
  <div class="thumbnail">
    <img src="inner-image.jpg" />
  </div>
</div>
