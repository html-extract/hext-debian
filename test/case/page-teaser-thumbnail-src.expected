{"image":"image.jpg"}
{"image":"inner-image.jpg"}
